server/index.js

```diff
@@ -0,0 +1,32 @@
+const Model = require('./db/model.js');
+const express = require('express');
+const app = express();
+const port = 3001;
+
+app.use(express.static(__dirname + '/../public/dist'));
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+
+app.get('/homedetails/:id', function (req, res) {
+  Model.find({listingId: req.params.id})
+    .then(data => {
+      res.status(200).send(JSON.stringify(data));
```

Comment:
If I recall, you don't have to stringify the data to send when you're using express.json()

Reply:
... you're probably right, I will check before I merge